ibm/data_source_ibm_is_lb_listeners_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMIsLbListenersDataSourceBasic(t *testing.T) {
+	vpcname := fmt.Sprintf("tflblis-vpc-%d", acctest.RandIntRange(10, 100))
+	subnetname := fmt.Sprintf("tflblis-subnet-%d", acctest.RandIntRange(10, 100))
+	lbname := fmt.Sprintf("tflblis%d", acctest.RandIntRange(10, 100))
+	protocol1 := "http"
+	port1 := "8080"
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMIsLbListenersDataSourceConfigBasic(vpcname, subnetname, ISZoneName, ISCIDR, lbname, port1, protocol1),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_is_lb_listeners.is_lb_listeners", "id"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_lb_listeners.is_lb_listeners", "lb"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_lb_listeners.is_lb_listeners", "listeners.#"),
```

Comment:
Could we please add the test support for the child attributes which are mandatory.